workers/api-server/src/http/wallet.rs

```diff
@@ -453,6 +471,12 @@ impl TypedHandler for CreateOrderHandler {
     ) -> Result<Self::Response, ApiServerError> {
         let wallet_id = parse_wallet_id_from_params(&params)?;
 
+        // Check that the order does not already exist
+        let oid = req.order.id;
+        if self.state.get_order_metadata(&oid).await?.is_some() {
```

Comment:
nit: should this just be `self.state.contains_order(&oid)`? semantically both of these queries should return `true` in the same cases, but just for readability & consistency w/ the check in the `AdminCreateOrderInMatchingPoolHandler`

Reply:
Good call